src/saml2/metadata.py

```diff
@@ -5,11 +5,7 @@
 from saml2.config import Config
 from saml2.validate import valid_instance
 from saml2.time_util import in_a_while
-from saml2.extension import mdui
-from saml2.extension import idpdisc
-from saml2.extension import shibmd
-from saml2.extension import mdattr
-from saml2.extension import sp_type
+from saml2.extension import mdui, idpdisc, shibmd, mdattr, sp_type, node_country
```

Comment:
One import per line is fine and simplifies the diff.

Reply:
Probably this Is Better

````
from x import (a,
                           B,
                           C)
````

Reply:
@c00kiemon5ter on import per line you mean the way @peppelinux suggests or the way it was before the diff?

Reply:
Following isort rules they should be in alphabetic order...
These are cosmetic things, great work John, as usual

Reply:
As it was before the diff.